### DISCUSSION TITLE
Discussions are opened as a way for users to, ask questions/help eachother, without opening an issue.. Please use the Q&A section for questions.

### DISCUSSION THREAD

**Discussion by illsk1lls** [illsk1lls/ZipRipper#18] [Announcements]
Note: Radeon support is experimental and may need additional work..

Feel free to comment or ask any questions related to the project in the Q&A section ;)

**Comment by replynow** [comment]
I have pdf with password with a pwd length of 10 to 24, Not sure what pwd is 
how much time it take ?
what is the best way 
Recommended  Computer or server I need to use if I want to open the file in 24 hrs
Can I use MacBook ?

**Reply by illsk1lls** [reply]
It is a Windows 10/11 based CMD script.

JohnTheRipper will probably run on Mac, it is used in linux natively

How long it takes depends on the password, give it a shot, it might not take too long.

**Comment by replynow** [comment]
Do u have any paid option

**Comment by illsk1lls** [comment]
no, none of my work will ever require payment, it is here free for all to use...

as far as the program goes, JohnTheRipper is the real star, ZipRipper is just a wrapper CMD script that helps fill in John's options as you go..

**Comment by replynow** [comment]
I am getting a message Passwords longer than 10 [worse case UTF - 8] to 32 [ASCII] rejectedBut I think my password is around 18 characters and mix of number, stumbles and letters. Is that will work ?ThanksOn Jun 5, 2024, at 9:37 AM, illsk1lls ***@***.***> wrote:﻿
no, none of my work will ever require payment, it is here free for all to use...
as far as the program goes, JohnTheRipper is the real star, ZipRipper is just a wrapper CMD script..

**Comment by jcrip1526** [comment]
Hello, I am trying to use the zipripper but everytime it will run and say it needs a moment. Then it will just close and not give me a password for my pdf file. What should I do?

**Comment by illsk1lls** [comment]
Please post questions in the Q&A section to keep things better organized.. For this one we can continue in the issues section on your other post since it may be a bug.

**Comment by requvium** [comment]
how to fix this error--

Running JohnTheRipper...

Device 1: NVIDIA GeForce 940MX
Using default input encoding: UTF-8
Loaded 8 password hashes with 8 different salts (ZIP-opencl, WinZip [PBKDF2-SHA1 OpenCL])
Loaded hashes with cost 1 (HMAC size) varying from 911527 to 37071403
Note: Passwords longer than 21 [worst case UTF-8] to 64 [ASCII] rejected
0: OpenCL CL_INVALID_COMMAND_QUEUE (-36) error in opencl_zip_fmt_plug.c:185 - Error releasing memory mappings

Resume is UNAVAILABLE for this file ;(
